src/__tests__/ext.spec.js

```diff
@@ -113,6 +113,21 @@ describe("ext", () => {
         },
       },
     ];
+    const chartIds = {
```

Comment:
same here, this isn't a list of chart id, it is a sheet object, that has a list of cells